util/encrypt.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package util
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	"github.com/marcopeereboom/sbox"
+)
+
+// VersionEncryptionKey is the version of the EncryptionKey struct.
+const VersionEncryptionKey = 1
+
+// EncryptionKey wraps a 32 byte encryption key.
+type EncryptionKey struct {
+	Version uint      `json:"version"` // Version of this struct
+	Key     *[32]byte `json:"key"`     // Key used for encryption
```

Comment:
After rereading this, this file needs to go.

Hex encode the key on disk.
When you load it and convert it from hex into a [32]byte slice you return the address to the caller.
Zero out in between copies of the key.
Store key as a pointer in the calling context.

Reply:
Done.